torchstain/torch/normalizers/macenko.py

```diff
@@ -49,7 +49,11 @@ def __find_concentration(self, OD, HE):
         Y = OD.T
 
         # determine concentrations of the individual stains
-        return torch.linalg.lstsq(HE, Y)[0]
+
+        if torch.__version__ >= (1,9,0):
```

Comment:
Great! It is a little suboptimal to have this if statement being called every single time this method is called. Might be better to do this only once in the __init__() or similar. And then, this method knows which lstsq to use, but we can fix that in the near future. LGTM.

Reply:
I agree, it should be a rather quick fix to do (in init or in the file header)